app/services/authorizer.rb

```diff
@@ -104,10 +104,18 @@ def build_filtered_scope_components(resource_class, all_filters, options)
     return result if all_filters.any?(&:unlimited?)
 
     search_string = build_scoped_search_condition(all_filters.select(&:limited?))
-    find_options = ScopedSearch::QueryBuilder.build_query(resource_class.scoped_search_definition, search_string, options)
-    result[:where] << find_options[:conditions]
-    result[:includes].push(*find_options[:include])
-    result[:joins].push(*find_options[:joins])
+
+    begin
+      find_options = ScopedSearch::QueryBuilder.build_query(resource_class.scoped_search_definition, search_string, options)
+
+      result[:where] << find_options[:conditions]
+      result[:includes].push(*find_options[:include])
+      result[:joins].push(*find_options[:joins])
+    rescue ScopedSearch::QueryNotSupported => e
+      Foreman::Logging.logger('permissions').error N_("Scoped search query not supported: #{e.message}")
```

Comment:
Perhaps late to the review since this is a cherry pick, but this is invalid use of translations. You can't use string interpolation in translations.

Typically we don't translate logging anyway so the easiest fix is:

```suggestion
      Foreman::Logging.logger('permissions').error "Scoped search query not supported: #{e.message}"
```

However, it's probably better to merge that in develop first and cherry pick that too. It's only marking it for translation now.

Reply:
https://github.com/theforeman/foreman/pull/10684

Reply:
Thanks! That's merged so please include it here :)